src/styles/utils.scss

```diff
@@ -0,0 +1,17 @@
+$day-color: #eee;
+$day-width: 100px;
+$day-height: 100px;
+$day-font: Arial;
+$day-font-size: 30px;
+$hover-color: #f0eaeb;
```

Comment:
Fix hover color as on the task description 
<img width="195" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/64750958/8e3ac391-6ec9-421a-980b-0e13a7e1e5b3">